{"ast":null,"code":"import { createErrorClass } from './createErrorClass';\nexport var SequenceError = createErrorClass(function (_super) {\n  return function SequenceErrorImpl(message) {\n    _super(this);\n\n    this.name = 'SequenceError';\n    this.message = message;\n  };\n});","map":{"version":3,"sources":["C:\\Users\\vicen\\Documents\\React\\visualizador\\node_modules\\rxjs\\src\\internal\\util\\SequenceError.ts"],"names":[],"mappings":"AAAA,SAAS,gBAAT,QAAiC,oBAAjC;AAoBA,OAAO,IAAM,aAAa,GAAsB,gBAAgB,CAC9D,UAAC,MAAD,EAAO;AACL,SAAA,SAAS,iBAAT,CAAsC,OAAtC,EAAqD;AACnD,IAAA,MAAM,CAAC,IAAD,CAAN;;AACA,SAAK,IAAL,GAAY,eAAZ;AACA,SAAK,OAAL,GAAe,OAAf;AACD,GAJD;AAIC,CAN2D,CAAzD","sourcesContent":["import { createErrorClass } from './createErrorClass';\n\nexport interface SequenceError extends Error {}\n\nexport interface SequenceErrorCtor {\n  /**\n   * @deprecated Internal implementation detail. Do not construct error instances.\n   * Cannot be tagged as internal: https://github.com/ReactiveX/rxjs/issues/6269\n   */\n  new (message: string): SequenceError;\n}\n\n/**\n * An error thrown when something is wrong with the sequence of\n * values arriving on the observable.\n *\n * @see {@link operators/single}\n *\n * @class SequenceError\n */\nexport const SequenceError: SequenceErrorCtor = createErrorClass(\n  (_super) =>\n    function SequenceErrorImpl(this: any, message: string) {\n      _super(this);\n      this.name = 'SequenceError';\n      this.message = message;\n    }\n);\n"]},"metadata":{},"sourceType":"module"}